suite-native/transactions/src/components/TransactionDetail/TransactionDetailSheets.tsx

```diff
@@ -27,11 +28,14 @@ const sheetToAnalyticsEventMap: Record<SheetType, TransactionSheetAnalyticsEvent
     inputs: EventType.TransactionDetailInputOutput,
 };
 
+const cardStyle = prepareNativeStyle(() => ({ paddingVertical: 12, paddingRight: 12 }));
```

Comment:
I see, we don't have a constant for 12 🙈

Reply:
Yeah, there is no 12 spacing const. But IMO we should add it. Because most of the new designs are based on values 8, 12 and 16. So we are hardcoding the value 12 too often. Let's discuss with other team devs on Slack.